example/src/screens/InitialScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import { BrandColors } from '@colors';
+import { BackgroundWrapper } from '@components/BackgroundWrapper';
+import { Logo } from '@components/Icon';
+import { Typo } from '@components/Typo';
+import { CardButton } from '@components/buttons/CardButton';
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+export const InitialScreen = ({ navigation }) => {
+  return (
+    <BackgroundWrapper>
+      <View style={styles.content}>
+        <View style={styles.header}>
+          <Logo
```

Comment:
You've put the whole logo as a font? Interesting :D Normally I'd just export it to png image

![Screenshot 2023-03-21 at 16 52 06](https://user-images.githubusercontent.com/6280457/226665108-15ded08c-b235-4184-9d19-792c449ff5d5.png)

Reply:
Is it a bad thing? There are smaller logos that are basically an icon without ant text, so I assumed it will be fine.

Reply:
I don't know :D I'd lean towards a png, because it's easier to use (just an Image component) and more conventional. In terms of performance I haven't measured it but I guess png might be faster, it shouldn't matter much. In terms of quality font would win but the difference shouldn't be significant, especially if you provide 3 sizes of the logo.